js/contact-firebase.js

```diff
@@ -0,0 +1,58 @@
+// Your web app's Firebase configuration
+var firebaseConfig = {
+    apiKey: "AIzaSyBWcZwrJrpVk1Wac6b1cbEdPPtOx-5Sbvs",
+    authDomain: "contact-us-page-36e86.firebaseapp.com",
+    databaseURL: "https://contact-us-page-36e86.firebaseio.com",
+    projectId: "contact-us-page-36e86",
+    storageBucket: "contact-us-page-36e86.appspot.com",
+    messagingSenderId: "817558500147",
+    //appId: "1:817558500147:web:b847921b48e3c270"
+};
+// Initialize Firebase
+firebase.initializeApp(firebaseConfig);
+var database = firebase.database();
+
+
+//Submit button for adding message - then update the database
+$("#submitButton").on("click", function (event) {
+    event.preventDefault();
+
+    // Grabs user input
+    var name = $("#name").val().trim();
+    var emailAdd = $("#email").val().trim();
+    var message = $("#message").val().trim();
+    var replied = "not yet";
+
+    // Creates local "temporary" object for holding train data
+    var newMsg = {
+        name: name,
+        emailAdd: emailAdd,
+        message: message,
+        replied: replied
+    };
+
+    alert("Your message has been successfully sent📤. We will get back to you soon. Thank you!");
+
+    // Clears all of the text-boxes
+    $("#name").val("");
+    $("#email").val("");
+    $("#message").val("");
+
+    // Uploads employee data to the database
+    database.ref().push(newMsg);
```

Comment:
You don't want the new Msg to be stored in the root of the database. it should be under the child's object found by querying the uid of the child. this can be found in the currentUser object once you include userInfo.js into contact.html.
We can talk about this tonight.
Great approach though

Reply:
Issue resolved. Thanks!